NEW_MODULE_TECH_EVAL.MD

```diff
@@ -34,14 +34,23 @@ This document outlines the processes for **technical** evaluation of module for
       * In such instances the Evaluator will contact the Submitter with a request for a demonstration, and a mutually suitable time for the demonstration will be scheduled.
 
 ### Submission form
-Information gathered includes:
+
+The submission form uses a Jira template.  To access the form:
+1. Open [Create issue from template](https://folio-org.atlassian.net/jira/software/projects/TCR/apps/11037d47-9c1c-48c5-91dc-17f8cd51d247/e758ec95-a2c9-44c8-b93e-36b2eb5d1716), also available in the sidebar of the [Jira Technical Council Requests](https://folio-org.atlassian.net/jira/software/c/projects/TCR) project.
+1. Select:
+    1. Issue type: Task
+    1. Template name: Request for Technical Evaluation of a Module (TEMP-17).
+1. Confirm.
```

Comment:
@dltj At TC today, none of us actually knew how to access this template for the new module evaluation form.  This process works, but is there any direct link or other better way?